docs/content/kubernetes-tutorial.md

```diff
@@ -196,7 +345,10 @@ spec:
             - "--tls-private-key-file=/certs/tls.key"
             - "--addr=0.0.0.0:8443"
             - "--addr=http://127.0.0.1:8181"
+            - "--set=services.default.url=http://host.minikube.internal:8888"
```

Comment:
Since we said earlier we'd recommend kind as well, might want to add a comment on how this can be done?
Another option would be to deploy the bundle nginx to the kubernetes cluster as that would work regardless of cluster type.

Reply:
Added a comment about kind.

Reply:
Great!